### ISSUE TITLE
CGAN: Reconstruction Loss?

### ISSUE THREAD

**Issue by ZakariaeBelmekki98** [aldelb/non-verbal-behaviours-generation#3]
Hi again. Just a quick question on the CGAN training. In page 8, the loss function is the typical adversarial loss, but in the code I can see what I suppose is a reconstruction loss (MSELoss) applied to the generated outputs. Would you please clear that up for me?  Cheers!

**Comment by ZakariaeBelmekki98** [comment]
Ah my bad, you don't backpropagate through that loss, therefore it is not included in the training.
